conanfile.py

```diff
@@ -3,7 +3,7 @@
 
 class NuRaftGrpcConan(ConanFile):
     name = "nuraft_grpc"
-    version = "6.0.5"
+    version = "6.0.6"
```

Comment:
Since this is a *new* API call we should bump the MINOR and reset the PATCH to `0`